stable/swagger/templates/ingress.yaml

```diff
@@ -0,0 +1,37 @@
+{{- if .Values.ingress.enabled -}}
+{{- if not .Values.ingress.oauth.enabled -}}
+{{- $swaggerName := include "swagger.fullname" . -}}
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: {{ $swaggerName }}
+  labels:
+    app: {{ template "swagger.name" . }}
+    chart: {{ template "swagger.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+  annotations:
+    {{- range $key, $value := .Values.ingress.annotations }}
+    {{ $key }}: {{ $value | quote }}
+    {{- end }}
+    {{- if .Values.ingress.basicAuth.enabled }}
```

Comment:
Remove this block. It's specific to nginx-ingress. Annotations are already configurable.